pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: flutter_blue
 description:
   Flutter plugin for connecting and communicating with Bluetooth Low Energy devices,
   on Android and iOS
-version: 0.6.0+4
+version: 0.6.1
```

Comment:
```suggestion
version: 0.6.0+4
```

Reply:
If I make this change do I remove the entry from the CHANGELOG.md too?

Reply:
I would do so.